plugin/ochttp/propagation/propagation.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package propagation contains HTTP propagators.
+package propagation
```

Comment:
Even though this is maybe the most "official" way to propagate tracing information, I still think we should put it in its own package rather than the root package. Having it here also strongly implies that it's the default when in fact B3 is the default.

Reply:
I moved it to tracecontext subdirectory. PTAL.